yalelibrary/blacklight-marc
# fix refworks

Close #2625 https://github.com/yalelibrary/search-frontend/issues/2625
Fix the refworks failed
To test:
-- git pull this branch to your local directory

Then go to search-frontend directory to test: 
add the blacklight-marc gem to vagrant local by add this to the Vagrantfile:
config.vm.synced_folder "/directory of your local path where blacklight-marc saved/blacklight-marc", "/home/vagrant/blacklight-marc"
update gemfile to point the blacklight-marc to local vagrant directory: gem 'blacklight-marc', path: '/home/vagrant/blacklight-marc'
bundle install
vagrant up
vagrant ssh search-frondend
rails s -p 3000
search books/ single book page, result page, archives space result list and save list to "Export to Refworks"
Expect the result to be exported to your refworks account.

Notes: 
1. the search result page endnote did not export successfully, endnote export works in the the single title display page.
I will create a separate ticket. 
2. Refworks export for the archival space records in the New Arrival result does not work. I will create a separate ticket.
